api/src/index.ts

```
@@ -78,245 +79,157 @@ const apiRouter = express.Router();
 const web3IndexRouter = express.Router();
 
 apiRouter.get("/templates", cache(60 * 5), async (req, res) => {
-  try {
-    const templates = await getTemplateGallery();
-    res.send(templates);
-  } catch (err) {
-    console.error(err);
-    res.status(500).send("Failed to fetch templates");
-  }
+  const templates = await getTemplateGallery();
```

We let the errors throw and get caught by a middleware?